cmd/kubeless/list.go

```diff
@@ -100,26 +100,33 @@ func doList(w io.Writer, tprClient rest.Interface, ns, output string, args []str
 func printFunctions(w io.Writer, functions []*spec.Function, output string) error {
 	if output == "" {
 		table := tablewriter.NewWriter(w)
-		table.SetHeader([]string{"Name", "namespace", "handler", "runtime", "type", "topic", "dependencies"})
+		table.SetHeader([]string{"Name", "description", "namespace", "handler", "runtime", "type", "topic", "dependencies", "memory", "env"})
 		for _, f := range functions {
 			n := f.Metadata.Name
+			desc := f.Spec.Desc
```

Comment:
here, switching to annotation would give us something like `f.Metadata.Annotation['description']`